test/test_cuda.py

```diff
@@ -3607,6 +3607,7 @@ def test_graph_memory_stats_and_use_result_after_destroy_graph(self):
             torch.cuda.synchronize()
             torch.cuda.empty_cache()
 
+    @skipIfRocm
```

Comment:
This is skipping a test case which was marked as passed for MS3.

Reply:
In other words, Doug identified two unit tests that could cause a corruption of hipStream that would result in a non-hipGraph-related test failing down the line. One of them (test_graph_grad_scaling) has been patched in this PR. The other one (test_graph_record_stream) is being skipped.